tests/test_redis_client.py

```diff
@@ -177,3 +179,9 @@ def test_should_not_call_rate_limit_if_not_enabled(mocked_redis_client, mocked_r
 
     assert not mocked_redis_client.exceeded_rate_limit('key', 100, 100)
     assert not mocked_redis_client.redis_store.pipeline.called
+
+
+def test_expire(mocked_redis_client):
```

Comment:
Add  some sort of test to check that the error message has the key and operation in it?